ui/package.json

```diff
@@ -3,23 +3,31 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
```

Comment:
Make sure to use fixed versions of dependencies.

Reply:
Done